### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Massimo37**
Initial commit for the Uno sample project (TreeView, DataGrid, TabView)

### CODE REVIEW

**Comment by Massimo37** [comment]
@jeromelaban  how can I fix the CI error:

https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=26409&view=logs&j=b0e9e703-bc7b-5b6a-2ec3-61742e5311eb&t=fe694602-11e4-517a-11ac-0f0951e3424d&l=34890

**Comment by jeromelaban** [comment]
Looks like something changed in the SkiaGtk sample that caused this. You're likely to have the same error locally.

File: UI/StatusBarThemeColor/StatusBarColorSample.Wasm/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (davidjohnoliver) [inline UI/StatusBarThemeColor/StatusBarColorSample.Wasm/Properties/launchSettings.json:1]: File unrelated to this PR?

**Review by davidjohnoliver — COMMENTED** [review]
Can you add a readme file with a short explanation of what's in the sample?

File: UI/StatusBarThemeColor/StatusBarColorSample.Wasm/Properties/launchSettings.json (hunk 1 above)
Comment (Massimo37) [reply UI/StatusBarThemeColor/StatusBarColorSample.Wasm/Properties/launchSettings.json]: Fixed.

**Comment by Massimo37** [comment]
@davidjohnoliver  Just added a simple readme that I plan to update as I progress.

**Comment by Xiaoy312** [comment]
> error NETSDK1136: The target platform must be set to Windows (usually by including '-windows' in the TargetFramework property) when using Windows Forms or WPF, or referencing projects or packages that do so.

The "CI" error, that also happens locally, was caused by the addition of `Prism.Wpf`.
Resolved by using `Prism.Uno` instead.

**Pull request merged**